css/style.css

```diff
@@ -0,0 +1,215 @@
+/* 
+Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+Font sizes (px)
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+*/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.main-wrapper {
+  width: 160rem;
+  margin: 0 auto;
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.promo-header {
+  position: absolute;
+  display: flex;  
+  height: 12rem;  
+  align-items: center;
+  padding-left: 18rem;    
+  gap: 0.8rem;
+}
+
+.header-shadow {
+  width: 160rem;
+  position: absolute;  
+  height: 12rem;    
+  background: linear-gradient(to bottom, black -100%, transparent 90%);  
+}
+
+.logo-menu-bar {
+  display: flex;
+  justify-content: space-between;
+  gap: 22.8rem;
+  align-items: center;
+}
```

Comment:
Redundant

Reply:
deleted. Changed the properties of the logo. Adding padding on the right side